app/models/source_manager.rb

```diff
@@ -0,0 +1,18 @@
+class SourceManager
+  class << self
+    PROTOCOLS = ['http', 'https']
```

Comment:
Hmm, not sure if I agree with hound on this one to be honest. Single quotes make it clean that no interpolation is required.